File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/ThreadResponse.kt
package com.sdkwork.ai

data class ThreadResponse(
    val id: String? = null,
    val object: String? = null,
    val metadata: Map<String, String>? = null,
    val createdAt: Int? = null,
    val toolResources: Map<String, Any>? = null
)

File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/thread_response.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class ThreadResponse:
    """线程响应"""
    id: str = None
    object: str = None
    metadata: Dict[str, str] = None
    created_at: int = None
    tool_resources: Dict[str, Any] = None
